tuned/plugins/plugin_video.py

```diff
@@ -124,3 +199,24 @@ def _get_radeon_powersave(self, device, ignore_missing = False):
 			return "dpm-" + self._cmd.read_file(sys_files["dpm_state"]).strip()
 		else:
 			return None
+
+	@command_set("panel_power_savings", per_device=True)
+	def _set_panel_power_savings(self, value, device, sim, remove):
+		"""Set the panel_power_savings value"""
+		try:
+			value = int(value, 10)
+		except ValueError:
+			log.warn("Invalid value %s for panel_power_savings" % value)
+			return None
+		if value in range(0, 5):
+			self.target_value = value
+			return self.apply_panel_power_saving_target(device, sim)
+		else:
+			log.warn("Invalid value %s for panel_power_savings" % value)
+		return None
+
+	@command_get("panel_power_savings")
+	def _get_panel_power_savings(self, device, ignore_missing=False):
+		"""Get the current panel_power_savings value"""
+		fname = self._files(device)["panel_power_savings"]
```

Comment:
Check that the file exists the same way as when attempting to set it. This is necessary for the `tuned-adm verify` command to succeed when the device does not support `panel_power_savings`.
```suggestion
		"""Get the current panel_power_savings value"""
		if not os.path.exists(self._files(device)["panel_power_savings"]):
			log.debug("panel_power_savings is not supported on '%s'" % device)
			return None
		fname = self._files(device)["panel_power_savings"]
```

Reply:
Thx, looks good to me.